Multisig/UI/UI Library/View Controllers/LoadableViewController/LoadableViewController.swift

```diff
@@ -44,8 +44,6 @@ class LoadableViewController: UIViewController {
                            dataErrorView.refreshControl,
                            tableView.refreshControl].compactMap { $0 }
 
-        showOnly(view: tableView)
```

Comment:
make a smoke test that other screens work, since this touches the core component for many other screens